extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -598,6 +600,97 @@ angular.module('oppia').component('svgFilenameEditor', {
         createPolygon();
       };
 
+      var makeCurveCircle = function(left, top) {
```

Comment:
The term "Curve Circle" seems confusing.. can we use the term "bezier control points" instead? Maybe add a comment explaining what it means.

Reply:
Done.